designer/server/src/test/scala/pl/touk/nussknacker/ui/api/UserApiSpec.scala

```diff
@@ -0,0 +1,103 @@
+package pl.touk.nussknacker.ui.api
+
+import io.restassured.RestAssured.given
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import org.hamcrest.Matchers.equalTo
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.test.{
+  NuRestAssureExtensions,
+  NuRestAssureMatchers,
+  PatientScalaFutures,
+  RestAssuredVerboseLogging
+}
+import pl.touk.nussknacker.ui.api.helpers.{NuItTest, NuScenarioConfigurationHelper, WithMockableDeploymentManager}
+
+class UserApiSpec
+    extends AnyFreeSpecLike
+    with NuItTest
+    with WithMockableDeploymentManager
+    with NuScenarioConfigurationHelper
+    with NuRestAssureExtensions
+    with NuRestAssureMatchers
+    with RestAssuredVerboseLogging
+    with PatientScalaFutures {
+
+  "The endpoint for getting user info when" - {
+    "authenticated should" - {
+      "return user info" in {
+        given()
+          .auth()
+          .basic("allpermuser", "allpermuser")
+          .when()
+          .get(s"$nuDesignerHttpAddress/api/user")
+          .Then()
+          .statusCode(200)
+          .body(
+            equalsJson(s"""{
+               |  "id": "allpermuser",
+               |  "username": "allpermuser",
+               |  "isAdmin": false,
+               |  "categories": [ "Category1" ],
+               |  "categoryPermissions": {
+               |      "Category1": [
```

Comment:
let's write it like that:
```
"Category1": [ "Deploy", "Read", "Write" ]
```

to save some space (Without losing the readablity)